collect_app/src/main/java/org/odk/collect/android/widgets/viewmodels/DateTimeViewModel.java

```diff
@@ -23,7 +24,9 @@
         setSelectedTime(hourOfDay, minuteOfHour);
     };
 
-    public LocalDateTime date;
+    public LocalDateTime localDateTime;
```

Comment:
You should be consequent here. Below you have private props accessed via getters, would be good to keep one approach.